docker/db/mysql/init/ddl.sql

```diff
@@ -0,0 +1,63 @@
+GRANT ALL PRIVILEGES ON *.* TO 'root'@'localhost';
+FLUSH PRIVILEGES;
+
+CREATE DATABASE IF NOT EXISTS `chess` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+CREATE DATABASE IF NOT EXISTS `chess_test` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci;
+
+USE chess;
+
+CREATE TABLE chess_board
+(
+    chess_board_id BIGINT      NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
    id BIGINT      NOT NULL AUTO_INCREMENT,
```
`chess_board`의 id인데 굳이 명시할 필요 없이 `id`로도 충분하지 않을까요?

Reply:
`id`로 충분할 것 같아 변경하였습니다!